src/contexts/SolanaWalletContext.tsx

```diff
@@ -49,15 +82,18 @@ export const SolanaWalletProvider: FC = (props) => {
     if (network) {
       wallets.push(new BloctoWalletAdapter({ network }));
     }
+
     return wallets;
-  }, []);
+  }, [isBrave]);
 
-  return (
+  return isBrave !== null ? (
     <ConnectionProvider endpoint={SOLANA_HOST}>
       <WalletProvider wallets={wallets} autoConnect>
         {props.children}
       </WalletProvider>
     </ConnectionProvider>
+  ) : (
```

Comment:
Probably the final result would be the same, but I personally think that empty ternaries are not good, for me something like this would look better:

```ts
if (isBrave === null) return null;

return (
  <ConnectionProvider endpoint={SOLANA_HOST}>
    ...
  </ConnectionProvider>
)
```

Reply:
Fixed ✅

Leave as is:
```

if (isBrave === null) return <></>;
```

returning `null` was causing some problems.